source/_integrations/template.markdown

```diff
@@ -2174,6 +2200,7 @@ vacuum:
     state:
       description: "Defines a template to get the state of the vacuum. Valid value: `docked`/`cleaning`/`idle`/`paused`/`returning`/`error`"
       required: false
+      default: optimistic
```

Comment:
What does optimistic as default mean?

Reply:
It's all over the template docs already.  It means that if you omit the field, that value will be optimistically set by actions on the entity.